ocaml/testsuite/tests/jane-modular-syntax/user_error3.compilers.reference

```diff
@@ -1,5 +1,5 @@
-File "user_error3.ml", line 21, characters 26-28:
-21 | let _misnamed_extension = () [@jane.erasable.this_extension_doesn't_exist];;
-                               ^^
-Error: Unknown extension "this_extension_doesn't_exist" referenced via
-       a [@jane.erasable.this_extension_doesn't_exist] attribute
+File "user_error3.ml", line 24, characters 2-4:
+24 |   ()
+       ^^
+Error: All attribute embeddings are expected to contain a location attribute,
```

Comment:
This doesn't seem like the behavior that the test is meant to exhibit. Could we update the test? (This may be solved by my suggestion to fall back to using the Jane Syntax node's location in the absence of the location attribute.)

Reply:
I left a comment about that -- because the location attribute is required to be ghost, there's no way to write these explicitly anymore.  That may or may not be a regression.  We should talk offline about this piece of the design.

Reply:
We discussed, and we decided that either we should use `*_loc_stack` (in which case we should delete this test) or we should take the approach of using the de-ghostified attribute location in anticipation of syntactic arity (in which case we can revert the change to this test and keep it around).

Reply:
We're using `loc_stack` now :-)